### PR TITLE
Add support for associating multiple image zip files with posts and retrieving images

### PR DESCRIPTION
**Pull request by jancona**

### CODE REVIEW

File: api/posts.go [hunk 1]
```diff
@@ -7,13 +7,16 @@ import (
...
 	"log"
+	"time"
 
 	"gocloud.dev/blob"
 	"gocloud.dev/pubsub"
 
 	"github.com/ourrootsorg/cms-server/model"
 )
 
+const ImagesPrefix = "/images/%d/"
```
Comment (DallanQ) [inline api/posts.go:18]: I thought you had decided to put images under the posts, so /posts/id/images/filename?

File: server/posts.go [hunk 2]
```diff
@@ -67,6 +68,36 @@ func (app App) GetPost(w http.ResponseWriter, req *http.Request) {
...
+// @tags posts
+// @id getPostImage
+// @Param id path integer true "Post ID"
+// @Param imageFile path string true "Image file path"
+// @success 307 {header} string
+// @failure 404 {object} api.Error "Not found"
+// @failure 500 {object} api.Error "Server error"
+// @Security OAuth2Implicit[cms,openid,profile,email]
+// @Security OAuth2AuthCode[cms,openid,profile,email]
+func (app App) GetPostImage(w http.ResponseWriter, req *http.Request) {
```
Comment (DallanQ) [inline server/posts.go:83]: I'll modify this and the corresponding API function to accept height and/or width query parameters so clients can request thumbnails. We'll check if the thumbnail exists already and if not generate it before returning. (I've found that thumbnail generation is pretty fast.)

File: api/posts.go (hunk 1 above)
Comment (DallanQ) [reply api/posts.go:18]: Totally fine where they are as well; I'm not asking for a change here.

Comment (jancona) [reply api/posts.go:18]: `ImagesPrefix` is the S3 prefix, so the S3 files are named `/images/<postID>/<path in ZIP file>`. The URL to retrieve them is `/posts/<postID>/images/<path in ZIP file>`. I can make the S3 path match the URL if you think that's valuable.

Comment (DallanQ) [reply api/posts.go:18]: No, I don't think so. That looks good

**Pull request merged**